pages/dashboard.js

```diff
@@ -71,6 +89,8 @@ export default function Dashboard() {
                           <div>Date: {interview.date}</div>
                           <div>Nerves: {interview.nerves}</div>
                           <div>Excitement: {interview.excitement}</div>
+                          <textarea defaultValue={interview.notes} onChange={onChange} />
```

Comment:
what happens when there are newlines added in the textarea? do they persist after they get saved?

Reply:
Yeah they're saved as newline characters

Reply:
dope!